opentelemetry-resource-detectors/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.2.0
```

Comment:
Please use `vNext` instead of an explict version. The actual version would be specified during release. As done here - https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry/CHANGELOG.md.